ios/Classes/PdftronFlutterPlugin.m

```diff
@@ -108,18 +108,50 @@ + (void)configureTabbedDocumentViewController:(PTTabbedDocumentViewController*)t
     }
 }
 
+- (bool)getBase64:(NSString*)config
+{
+    if(config && ![config isEqualToString:@"null"])
+    {
+        //convert from json to dict
+        NSData* jsonData = [config dataUsingEncoding:NSUTF8StringEncoding];
+        id foundationObject = [NSJSONSerialization JSONObjectWithData:jsonData options:kNilOptions error:Nil];
+        
+        NSAssert([foundationObject isKindOfClass:[NSDictionary class]], @"config JSON object not in expected dictionary format.");
+        
+        if([foundationObject isKindOfClass:[NSDictionary class]])
+        {
+            NSDictionary* configPairs = (NSDictionary*)foundationObject;
+            id isBase64 = configPairs[PTIsBase64Key];
+            
+            if ([isBase64 isKindOfClass:[NSNumber class]]) {
+                return [isBase64 boolValue];
+            }
+        }
+        else
+        {
+            NSLog(@"config JSON object not in expected dictionary format.");
+        }
+    }
+    return NO;
```

Comment:
Probably better to include an error parameter as part of this method, and set it if there is an error. So the signature would be 

```- (bool)getBase64:(NSString*)config error:(NSError**)error```
And you would assign the error parameter to something if there was an error, so that it can programmatically be determined if there was an error.

The calling function would look like this:

```
NSError* error;

bool is64 = [myPlugin getBase64:config error:&error];
if( error != NULL )
{
  // handle error
  NSLog(@"Oh no there was an error: %@", error.description);
}